apps/src/templates/hintsDisplay.jsx

```diff
@@ -0,0 +1,57 @@
+module.exports = React.createClass({
+
+  propTypes: {
+    hintReviewTitle: React.PropTypes.string.isRequired,
+    getSeenHints: React.PropTypes.func.isRequired,
+    getUnseenHints: React.PropTypes.func.isRequired,
+    recordUserViewedHint: React.PropTypes.func.isRequired,
+    lightbulbSVG: React.PropTypes.node.isRequired,
+  },
+
+  getInitialState: function () {
+    return {
+      seenHints: this.props.getSeenHints(),
+      unseenHints: this.props.getUnseenHints(),
```

Comment:
Why not pass in the results of these funcs as props instead of passing in the funcs and storing the result as state?